bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckRun.java

```diff
@@ -328,24 +323,25 @@ private Map<String, String> blockingIntegrationLabels() {
         return Map.of("rejected", "The change is currently blocked from integration by a rejection.");
     }
 
-    private List<String> botSpecificIntegrationBlockers(List<Issue> issues) {
+    private List<String> botSpecificIntegrationBlockers(Map<Issue, Optional<IssueTrackerIssue>> issues) {
         var ret = new ArrayList<String>();
 
         var issueProject = issueProject();
         if (issueProject != null) {
```

Comment:
I'm not completely sure, but if we don't, there is a risk that we change behavior when a bot is configured without an issue tracker/project. It would be a good followup to really investigate if that should be a valid configuration and how that is supposed to work.

Reply:
Sure. It's no harm to keep the check here.